Octokit/Helpers/ApiUrls.cs

```diff
@@ -556,60 +556,60 @@ public static Uri Milestones(string owner, string name)
         /// Returns the <see cref="Uri"/> that returns the specified label.
         /// </summary>
         /// <param name="owner">The owner of the repository</param>
-        /// <param name="repo">The name of the repository</param>
-        /// <param name="name">The milestone number</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="milestoneNumber">The milestone number</param>
```

Comment:
@shiftkey, yes, now parameter has name "labelName" in Label method.Here is commit 		0da6f0e